modules/sw360/pom.xml

```diff
@@ -159,6 +159,10 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why do you need the simple slf4j backend implementation? Is this a test dependency?

Reply:
This gets rid of the warning of the missing SLFJ4 warning when executing the tests. 
I can remove this though and do it in a separate pull request.

Reply:
So you could add it as a testing dependency?

Reply:
I think this occurs when executing the classes in general, it just is visible when executing the tests.
I'd rather check this out later again and remove the change for now, if you don't mind.